proto/services/task_svc/v1/patient_svc.proto

```diff
@@ -244,4 +258,5 @@ service PatientService {
   rpc GetPatientDetails(GetPatientDetailsRequest) returns (GetPatientDetailsResponse) {}
   rpc GetPatientList(GetPatientListRequest) returns (GetPatientListResponse) {}
   rpc DeletePatient(DeletePatientRequest) returns (DeletePatientResponse) {}
+  rpc ReactivatePatient(ReactivatePatientRequest) returns (ReactivatePatientResponse) {}
```

Comment:
not the biggest fan of the name, `AdmitPatient` maybe?

Reply:
Me neither, maybe we should ask @aegis301

Reply:
Maybe Readmittance?